ui/src/app/modules/what-if/services/app-profile.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { AppProfile } from '../components/app-profile';
+import { BACKEND_URL } from '../../../app.component'
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+}
+)
+export class AppProfileService {
+  appProfile : AppProfile;
+  scenarioName : string;
+  url_submit_spec : string;
+
+  constructor(private http: HttpClient) {
+    //this.url_submit_spec = BACKEND_URL  + 'submit/whatif'
+    this.url_submit_spec = 'http://localhost:3030/api/submit/whatif'
```

Comment:
use the commented line to define URLs so that all URLs will point to a particular backend